src/script/components/list/ConversationListCallingCell.tsx

```diff
@@ -0,0 +1,378 @@
+                  {showVideoButton && (
+                    <button
+                      className={cx('call-ui__button', {'call-ui__button--active': selfSharesCamera})}
+                      onClick={() => callActions.toggleCamera(call)}
+                      disabled={disableVideoButton}
+                      data-uie-name="do-toggle-video"
+                      title={t('videoCallOverlayVideo')}
+                      data-uie-value={selfSharesCamera ? 'active' : 'inactive'}
+                    >
+                      <Icon.Camera className="small-icon" />
+                    </button>
+                  )}
+                  {isOngoing && (
+                    <div
+                      className={cx('call-ui__button', {
+                        'call-ui__button--active': selfSharesScreen,
+                        'call-ui__button--disabled': disableScreenButton,
+                        'with-tooltip with-tooltip--bottom': disableScreenButton,
+                      })}
+                      data-tooltip={disableScreenButton ? t('videoCallScreenShareNotSupported') : undefined}
+                      onClick={() => callActions.toggleScreenshare(call)}
+                      data-uie-name="do-call-controls-toggle-screenshare"
+                      data-uie-value={selfSharesScreen ? 'active' : 'inactive'}
+                      data-uie-enabled={disableScreenButton ? 'false' : 'true'}
+                      title={t('videoCallOverlayShareScreen')}
+                    >
+                      <Icon.Screenshare className="small-icon" />
+                    </div>
+                  )}
+                </div>
+
+                <div className="conversation-list-calling-cell-controls-right">
+                  {showParticipantsButton && (
+                    <div
+                      className={cx('call-ui__button call-ui__button--participants', {
+                        'call-ui__button--active': showParticipants,
+                      })}
+                      onClick={() => setShowParticipants(current => !showParticipants)}
+                      data-uie-name="do-toggle-participants"
+                    >
+                      <span>{t('callParticipants', participants.length)}</span>
+                      <Icon.Chevron className="chevron" />
+                    </div>
+                  )}
+                  {(isIncoming || isOutgoing) && (
+                    <div
+                      className="call-ui__button call-ui__button--red call-ui__button--large"
+                      onClick={() => (isIncoming ? callActions.reject(call) : callActions.leave(call))}
+                      title={t('videoCallOverlayHangUp')}
+                      data-uie-name="do-call-controls-call-decline"
+                    >
+                      <Icon.Hangup className="small-icon" style={{maxWidth: 17}} />
+                    </div>
+                  )}
+                  {isIncoming && !isDeclined && (
```

Comment:
We already checked for `!isDeclined`

```suggestion
                  {isIncoming && (
```